.github/ISSUE_TEMPLATE/GITHUB_ADMIN.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Astropy Github Organisation Administration
+about: Request that an Astropy GitHub org admin performs some action.
```

Comment:
This is too cryptic. Somewhere in this template needs some explanation on when one should and should not invoke this.

Reply:
Yeah sure, the scope is "things that can only be done by a GitHub org admin", suggestions how to word this would be welcome.

Reply:
How about some examples?